response_operations_ui/controllers/party_controller.py

```diff
@@ -125,6 +130,48 @@ def search_respondent_by_email(email):
     return response.json()
 
 
+def search_respondents(first_name, last_name, email_address, page=0):
+    if bool(os.environ.get('FAKE_PARTY_DATA', False)):
+        logger.debug('Respondent Search: Using fake data for controller')
+        data = mock_search_respondents(first_name, last_name, email_address, page)
+    else:
+        url = _get_search_respondents_url(first_name=first_name,
+                                          last_name=last_name,
+                                          email_address=email_address,
+                                          page=page)
+        response = requests.get(url, auth=app.config['PARTY_AUTH'])
+        if response.status_code != 200:
+            raise SearchRespondentsException(response,
+                                             first_name=first_name,
+                                             last_name=last_name,
+                                             email_address=email_address,
+                                             page=page)
+        data = response.json()
+
+    return data
+
+
+def _get_search_respondents_url(**kwargs):
+    url = f'{app.config["PARTY_URL"]}/party-api/v1/respondents?'
+
+    # Dict provides translation of variable names in response ops to url param names in party
+    kw_to_url_params = {
+        'first_name': 'firstName',
+        'last_name': 'lastName',
+        'email_address': 'email',
+        'page': 'page'
+    }
+
+    for resops_name, party_name in kw_to_url_params.items():
+        arg = kwargs.get(resops_name)
+        if arg and arg != '':
```

Comment:
I don't think you need the `and arg != ''` part because `if arg` will be truthy if any value is passed in and if empty it should skip.

Reply:
Done in next push